spotirec.py

```diff
@@ -307,24 +310,24 @@ def remove_from_blacklist(entries: list):
         with open(blacklist_path, 'r') as file:
             blacklist = json.loads(file.read())
     except json.decoder.JSONDecodeError:
-        print('Error: blacklist is empty')
+        logging.error('Error: blacklist is empty')
         exit(1)
     for uri in entries:
         if re.match(uri_re, uri):
             try:
                 try:
-                    print(f'Removing track {blacklist["tracks"][uri]["name"]} by '
+                    logging.info(f'Removing track {blacklist["tracks"][uri]["name"]} by '
                           f'{", ".join(str(x) for x in blacklist["tracks"][uri]["artists"]).strip(", ")} from blacklist')
                 except KeyError:
-                    print(f'Removing artist \"{blacklist["artists"][uri]["name"]}\" from blacklist')
+                    logging.info(f'Removing artist \"{blacklist["artists"][uri]["name"]}\" from blacklist')
                 del blacklist[f'{uri.split(":")[1]}s'][uri]
             except KeyError:
-                print(f'uri \"{uri}\" does not exist in your blacklist')
+                logging.error(f'uri \"{uri}\" does not exist in your blacklist')
                 # FIXME: Remove this notice at some point
-                print('Blacklist structure was recently re-done, so you may need to remove and re-do your blacklist. '
+                logging.info('Blacklist structure was recently re-done, so you may need to remove and re-do your blacklist. '
```

Comment:
325 should probably be a warning, as it doesn't exit. Also, you could probably remove the notice about blacklist restructuring by now.